Sources/Sentry/Public/SentrySDK.h

```diff
@@ -25,7 +25,8 @@ SENTRY_NO_INIT
 /**
  * Inits and configures Sentry (SentryHub, SentryClient) and sets up all integrations.
  */
-+ (void)startWithOptions:(NSDictionary<NSString *, id> *)optionsDict NS_SWIFT_NAME(start(options:));
++ (void)startWithOptions:(NSDictionary<NSString *, id> *)optionsDict
+    NS_SWIFT_NAME(start(options:));
```

Comment:
`l`: Do you maybe have different code formatting rules? `make format` should do the job. Can we maybe exclude these changes from the PR?

Reply:
I did, and it looks like this was an automated commit: https://github.com/getsentry/sentry-cocoa/pull/1071/commits/81523f17930128e8adf970ea413a15c17ba097f4